lib/hammer_cli_katello/content_export_helper.rb

```diff
@@ -17,6 +21,10 @@ def execute
       end
     end
 
+    def send_request
```

Comment:
shouldn't be necessary. we are overriding task_progress  itself to make it store ```@task``` so no need for the send_request to do the same

Reply:
I don't think task_progress is called from generate-metadata

Reply:
still not needed I think... Lemme test with it and without and tell ya :).

Reply:
Ok I am wrong.. Remove the task_progress and keep this guy intact